powerline/renderers/bar.py

```diff
@@ -1,45 +0,0 @@
-# vim:fileencoding=utf-8:noet
-from __future__ import (unicode_literals, division, absolute_import, print_function)
-
-from powerline.renderer import Renderer
-from powerline.colorscheme import ATTR_UNDERLINE
-
-
-class BarRenderer(Renderer):
-	'''bar (bar ain't recursive) renderer
-
-
-	See documentation of `bar <https://github.com/LemonBoy/bar>`_ and :ref:`the usage instructions <bar-usage>`
-	'''
-
-	character_translations = Renderer.character_translations.copy()
-	character_translations[ord('%')] = '%%'
```

Comment:
@S0lll0s Why do you think it is not needed? Literal `%{r}` is a *valid* segment contents and if you do not escape `%` it will be confused with `%{r}` that starts right side in place of displaying it literally.